app/src/renderer/components/wallet/PageSend.vue

```diff
@@ -121,9 +121,10 @@ export default {
     },
     ...mapActions(['walletSend'])
   },
+  props: ['denom'],
   mounted () {
-    if (this.denominations.length === 1) {
-      this.fields.denom = this.denominations[0].value
+    if (this.denom) {
```

Comment:
I set the denom from the property here. So the variable "fields.denom" stays the one truth for the currently selected denom.

Reply:
i think this is better. thanks for the improvement.